lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:cboard_mobile/Settings/ThemeManager.dart';
+import 'package:cboard_mobile/Settings/routes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'routes.dart';
+
+class Setting_wrapper extends StatelessWidget {
+  // ThemeNotifier theme = new ThemeNotifier();
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primaryColor: Color(0xFF23036A),
+        accentColor: Color(0xFFE3CAFF),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        scaffoldBackgroundColor: Colors.white,
+      ),
+      home: Scaffold(
+        backgroundColor: Color(0xffE5E5E5),
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text('Settings'),
+        ),
+        body: MyStatefulWidget(),
+      ),
+      routes: getRoute(),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
+  final sections = ['People', 'Language', 'System', 'Help'];
+  final List<Map<String, List<Object>>> features = <Map<String, List<Object>>>[
```

Comment:
Instead of using a general `List<Object>` here, what would be really cool if we made some `SettingListItem` class to standardize what's included here.

Reply:
Could you explain more on this ?

Reply:
Sure! So, in this Object, we have something like `List<Map<String<List<Object>>>`, which can be somewhat confusing to follow along with, because we don't really know what `Object` is. What you can do instead is make a class called `SettingListItem` to replace that generic Object instead - that class would have all the properties of the `Object` that you defined. I think I made an `Error` class that followed along with this, so that should give you a better idea of what that might look like!

Reply:
I get it. That's actually a good idea. I've implemented it. Probably I'll improve it for big sections like `People`, `Language` too